omod/src/main/java/org/openmrs/module/patientdashboard/web/controller/autocomplete/AutoCompleteController.java

```diff
@@ -362,7 +362,7 @@ public String detailClinical(@RequestParam(value="id",required=false) Integer id
 			
 			Date birthday = patient.getBirthdate();
 			model.addAttribute("age", PatientUtils.estimateAge(birthday));
-			model.addAttribute("ageCategory", PatientUtils.estimateAgeCategory(birthday));
+			//model.addAttribute("ageCategory", PatientUtils.estimateAgeCategory(birthday));

```

Comment:
Same here, we are NOT changing the work flow of the module, we need to maintain and only change things that are stopping it from being accessed on our UI. if there are metadata missing, we need to add them using the ehrconfigs projects

Reply:
commented it out to work on it later